Emma/emma.py

```diff
@@ -181,17 +187,24 @@ def processArguments(arguments):
     # Get paths straight (only forward slashes) or set it to empty if it was empty
     subDir = Emma.shared_libs.emma_helper.joinPath(arguments.subdir) if arguments.subdir is not None else ""
 
+    if arguments.memVis and arguments.memVisResolved:
+        sc().error("Select either memVis or memVisResolved")
+    if arguments.memVisResolved and arguments.noResolveOverlap:
+        sc().error("View of resolved overlaps is not possible as noResolveOverlap is active")
```

Comment:
I think this is a warning, not an error since we would present the unresolved figure.